pyproject.toml

```diff
@@ -1,38 +0,0 @@
-[tool.mypy]
```

Comment:
Removing this file is why `pre-commit` is failing. It need the `mypy` settings at this level.

Reply:
Do you know how this line was removed? Was it a particular step in the uv workflow that caused it to change?

Reply:
No reason I can think of that uv would remove it since uv itself uses the file. The only thing I can think of is maybe because we are only using it for settings and not project related it removed it.